assginment7/src/models/Student.java

```diff
@@ -0,0 +1,49 @@
+package models;
+
+public class Student {
+    private int id;
```

Comment:
Implements  with Serializable and could you let me know why we need implements "Serializable" interface ?